pkg/payload/preflight/clusterversion/upgrdeable.go

```diff
@@ -0,0 +1,57 @@
+package clusterversion
```

Comment:
name of this file is a typo

Reply:
done with https://github.com/openshift/cluster-version-operator/pull/243/commits/64249a4e283ddf99631885faedf6297401fb7e88